### ISSUE TITLE
Login page color scheme suggestion

### ISSUE THREAD

**Issue by pvanlaet** [openmotics/gateway-frontend#17]
Personally I believe it would look a bit more modern if the background of the login page was completely white.
Before:
![image](https://cloud.githubusercontent.com/assets/7603855/20903758/eb1527f2-bb3c-11e6-84b7-974fd6258f00.png)

After:
![image](https://cloud.githubusercontent.com/assets/7603855/20903752/e40f924e-bb3c-11e6-8e36-5e1496cffea7.png)

**Comment by jamipraveeen** [comment]
@pvanlaet If you dont mind can you provide installation steps (windows) apart from github documentation.Thank you

**Comment by khenderick** [comment]
@jamipraveeen, I'm not sure if I understand your question correctly. 

I understand that you want to know how to install this interface on Windows. However, the OpenMotics Gateway frontend is just the web interface running on the OpenMotics Gateway hardware. It is not written to run on anything else except the module itself. You can either buy this hardware from our webshop, or build it yourself using the hardware schematics.

**Comment by jamipraveeen** [comment]
@khenderick  Yes,I need to run only the interface But, I got error like this Please give me a solution.
![nodeerrr](https://cloud.githubusercontent.com/assets/20150985/22116556/d8afff74-de96-11e6-9e2d-d84e4f4018e7.jpg)

**Comment by khenderick** [comment]
Can you please create a separate ticket? This is not related to whether or not the login page's color scheme should be changed.
